frontend/src/components/CreateEvent/CreateEvent.jsx

```diff
@@ -1,10 +1,90 @@
 import "./CreateEvent.css"
+import { useState } from "react";
 
-function CreateEvent () {
+function CreateEvent ({isOpen, onClose, onSubmit}) {
+    const [title, setTitle]  = useState('');;
+    const [description, setDescription] = useState('');
+    const [startDate, setStartDate] = useState('');
+    const [startTime, setStartTime] = useState('');
+    const [endDate, setEndDate] = useState('');
```

Comment:
Do we need a start and end date? Should we handle a single date for an event for now?